VirtoCommerce.CatalogModule.Data/Search/ProductSearchService.cs

```diff
@@ -45,71 +45,30 @@ protected virtual ItemResponseGroup GetResponseGroup(ProductSearchCriteria crite
 
         protected override void ReduceSearchResults(IEnumerable<Product> products, ProductSearchCriteria criteria)
         {
-            var responseGroup = GetResponseGroup(criteria);
 
             foreach (var product in products)
             {
-                if (!responseGroup.HasFlag(ItemResponseGroup.ItemAssets))
-                {
-                    product.Assets = null;
-                }
-
-                if (!responseGroup.HasFlag(ItemResponseGroup.ItemProperties))
-                {
-                    product.Properties = null;
-                }
-                else if(!string.IsNullOrEmpty(criteria.LanguageCode))
+                
```

Comment:
Shouldn't we call product.ReduceDetails here and in CategorySearchService?

Reply:
In this place product type is Web.Models.Product and haven't got ReduceDetails method. By using toWebModel method we reducing details.